tests/README.md

```
@@ -122,8 +122,15 @@ For details, please have a look at the [benchmark guide](https://github.com/Sema
 
 # JavaScript (QUnit)
 
-Running qunit tests in connection with MediaWiki requires to execute
-[Special:JavaScriptTest][mw-qunit-testing].
+JavaScript tests live in `tests/node-qunit/` and run under Node.js with
+[QUnit](https://qunitjs.com/) in a jsdom + jQuery environment (see
+`tests/node-qunit/setup.js`), so they execute in CI without a MediaWiki or
+browser stack. Run them with:
+
+    npm install
```

markdownlint-cli2 (0.23.0)

[warning] Code block style
Expected: fenced; Actual: indented

(MD046, code-block-style)